clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -385,13 +383,14 @@ private FilesetContextPair getFilesetContext(Path virtualPath, FilesetDataOperat
             scheme,
             str -> {
               try {
-                Map<String, String> maps = getConfigMap(getConf());
                 FileSystemProvider provider = fileSystemProvidersMap.get(scheme);
                 if (provider == null) {
                   throw new GravitinoRuntimeException(
                       "Unsupported file system scheme: %s for %s.",
                       scheme, GravitinoVirtualFileSystemConfiguration.GVFS_SCHEME);
                 }
+
+                Map<String, String> maps = getConfigMap(getConf());
```

Comment:
What is the gvfs configuration for now, is it aligned with server side conf, or still use "."?

Reply:
They are the same as those in Gravitino server side.

Reply:
Have you verified manually using gvfs?

Reply:
Yes , `GravitinoVirtualFileSystemS3IT` can be running in the CI and cover the changes.